apps/prairielearn/assets/scripts/studentAssessmentClient.ts

```diff
@@ -0,0 +1,21 @@
+import { onDocumentReady } from '@prairielearn/browser-utils';
+
+onDocumentReady(() => {
+  const pledgeCertifyCheck = document.querySelector<HTMLInputElement>('#certify-pledge');
+  const startAssessmentForm = document.querySelector<HTMLFormElement>('#confirm-form');
+  const startAssessmentButton =
+    startAssessmentForm?.querySelector<HTMLButtonElement>('#start-assessment');
+
+  if (startAssessmentButton && pledgeCertifyCheck) {
+    pledgeCertifyCheck.addEventListener('change', () => {
+      startAssessmentButton.disabled = !pledgeCertifyCheck.checked;
+    });
+  }
+  if (startAssessmentButton && startAssessmentForm) {
+    startAssessmentForm?.addEventListener('submit', () => {
```

Comment:
The `if` statement above already checked if this is truthy.

```suggestion
    startAssessmentForm.addEventListener('submit', () => {
```